### PR TITLE
Create a product card

### PR DESCRIPTION
**Pull request by SashaViun**
- [DEMO LINK](https://SashaViun.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://SashaViun.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/product-card.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.product-card__cash {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
```
Comment (nkondd) [inline src/styles/blocks/product-card.css:59]:
```suggestion
  color: var(--color-dark);
```

File: src/styles/blocks/product-card.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  height: 40px;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+  line-height: 18px;
+  background-color: #00acdc;
```
Comment (nkondd) [inline src/styles/blocks/product-card.css:73]:
```suggestion
  background-color: var(--color-blue);
```

File: src/styles/blocks/product-card.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  border-radius: 5px;
+  border: 1px solid transparent;
+}
+
+.product-card__button:hover {
+  border-color: #00acdc;
+  color: #00acdc;
```
Comment (nkondd) [inline src/styles/blocks/product-card.css:80]:
```suggestion
  border-color: var(--color-blue);
  color: var(--color-blue);
```

File: src/styles/blocks/product-card.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+.product-card__button {
+  display: flex;
+  height: 40px;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
```
Comment (nkondd) [inline src/styles/blocks/product-card.css:70]:
```suggestion
  color: var(--color-white);
```

**Review by nkondd — CHANGES_REQUESTED** [review]
Good Job! 
It would be better to add some variables for colors.

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,53 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
```
Comment (Denchepornyuk) [inline src/index.html:13]: it's better to use article tag for card instead of div

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**